docs/api/extensions/text-direction.md

```diff
@@ -0,0 +1,77 @@
+---
+description: Left, right, center, whatever! Align the text however you like.
+icon: align-left
+---
+
+# TextDirection
+
+[![Version](https://img.shields.io/npm/v/@tiptap/extension-text-direction.svg?label=version)](https://www.npmjs.com/package/@tiptap/extension-text-direction)
+[![Downloads](https://img.shields.io/npm/dm/@tiptap/extension-text-direction.svg)](https://npmcharts.com/compare/@tiptap/extension-text-direction?minimal=true)
+
+This plugin automatically adds the `dir` attribute to a specified list of nodes. The direction is automatically detected based on the node's content.
+
+## Installation
+
+```bash
+npm install @tiptap/extension-text-direction
+```
+
+## Settings
+
+### types
+
+A list of nodes where the `dir` attribute should be added to. Usually something like `['heading', 'paragraph']`.
+
+Default: `[]`
+
+```js
+TextDirection.configure({
+  types: ["heading", "paragraph"],
+});
+```
+
+### directions
+
+A list of available options for the 'dir' attribute.
+
+Default: `['ltr', 'rtl']`
+
+```js
+TextDirection.configure({
+  directions: ["ltr", "rtl"],
+});
+```
+
+### defaultDirection
+
+The default direction.
+
+:::warning
+If you specify a defaultDirection, the `dir` attribute will be excluded in nodes that have the same direction as the defaultDirection.
+:::
+
+Default: `null`
+
+```js
+TextDirection.configure({
+  defaultDirection: "rtl",
+});
+```
```

Comment:
Hmm, maybe ignore is a better name for that. The idea is that some apps are designed for a specific language, let's say Arabic, so it's not necessary to add dir="rtl" to every node because that's the default direction that is set in the html tag.